src/blog/2024-10-13/index.md

```diff
@@ -0,0 +1,117 @@
+title: Pre-Alpha v9.0 Release Note
+summary: Release note for Neotokyo; Rebuild Pre-Alpha v9.0
+author: nullsystem
+
+# Pre-Alpha v9.0 Release Note
+2024-10-13 nullsystem
+
+* [Download v9.0-prealpha build](https://github.com/NeotokyoRebuild/neo/releases/tag/v9.0-prealpha)
+* [Install NT;RE (client)](/guide/install/)
+* [GitHub Issues (Bug reports and feature requests)](https://github.com/NeotokyoRebuild/neo/issues)
+
+## Known issue
+
+* Player collision has been broken in v9.0-prealpha due to attemping enemy collision, wait for v9.1-prealpha release for the fix
+
+## New features
+
+### VIP and TDM gamemodes
+VIP mode turns CTG capture points into VIP escape points and a random person is selected as a VIP.
+A VIP objective is to escape to the escape points while teammates tries to protect the
+VIP. The enemy instantly wins the round if the VIP gets killed. The VIP have a different player model, stats,
+and is limited to the SMAC (VIP exlusive weapon).
+
+TDM mode turns off both CTG capture points and ghosts, but enables respawning and round time extended to 10.25m.
```

Comment:
but enables respawning, and round time is extended

Oxford comma is optional https://www.grammarbook.com/blog/commas/oxford-comma/